src/main/java/com/t13g2/forum/storage/forum/IEvent.java

```diff
@@ -0,0 +1,7 @@
+package com.t13g2.forum.storage.forum;
+
+/**
+ *
+ */
+public interface IEvent {
+}
```

Comment:
What might this be for? If it's to specify an event interface you could just extend from Java's existing `Event`

Reply:
Okay will take a look at `Event`